modules/ossm-installing-cert-manager.adoc

```diff
@@ -70,36 +89,102 @@ metadata:
 spec:
   ca:
     secretName: istio-ca
----
 ----
-+
+
 .. Create the objects by running the following command:
 +
 [source, terminal]
 +
 ----
 $ oc apply -f issuer.yaml
 ----
-+
+
 .. Wait for the `istio-ca` certificate to contain the "Ready" status condition by running the following command:
 +
 [source, terminal]
 ----
 $ oc wait --for=condition=Ready certificates/istio-ca -n istio-system
 ----
 
-. Copy the `istio-ca` certificate to the `cert-manager` namespace so it can be used by istio-csr:
+. Create the `IstioCSR` custom resource:
+
+.. Create the `IstioCSR` custom resource similar to the following example:
++
+.Example `istioCSR.yaml` file
+[source, yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: IstioCSR
+metadata:
+  name: default
+  namespace: istio-csr
+spec:
+  istioCSRConfig:
+    certManager:
+      issuerRef:
+        name: istio-ca
+        kind: Issuer
+        group: cert-manager.io
+    istiodTLSConfig:
+      trustDomain: cluster.local
+    istio:
+      namespace: istio-system
+----
+
+.. Create the `istio-csr` agent by by running the following command:
 +
-.. Copy the secret to a local file by running the following command:
+[source, terminal]
++
+----
+$ oc create -f istioCSR.yaml
+----
+
+.. Verify if the `istio-csr` deployment is ready by running the following command:
 +
 [source, terminal]
++
+----
+$ oc get deployment -n istio-csr
+----
+
+. Install the `istio` resource:
++
+[NOTE]
+====
+The configuration disables the built-in CA server for {istio} and forwards certificate signing requests from `istiod` to `istio-csr`. The `istio-csr` agent obtains certificates for both `istiod` and mesh workloads from cert-manager. The istiod TLS certificate generated by `istio-csr` is mounted into the pod at a known location for use.
```

Comment:
```suggestion
The configuration disables the built-in CA server for {istio} and forwards certificate signing requests from `istiod` to the `istio-csr` agent. The `istio-csr` agent obtains certificates for both `istiod` and mesh workloads from the cert-manager Operator. The `istiod` TLS certificate that is generated by the `istio-csr` agent is mounted into the pod at a known location for use.
```
- For clarity, be sure to include the appropriate noun after you reference a resource, variable, parameter, object, etc. I'm not sure what to suggest to add after the term `istiod`.
- Will the intended audience for this procedure know where the "known location for use" is?